### PR TITLE
AWS pricing cache

### PR DESCRIPTION
**Pull request by sergeymirzoyan1991**

### CODE REVIEW

File: src/adapters/aws/clients/aws-catalog-client.ts [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+  static async collectAllPricingLists (
+    accountId: string,
+    resources: Response<Ebs | Ec2 | Eip | Rds | Elb>[],
+    credentialProvider: CredentialProvider,
+    pricingFallbackInterface?: PricingInterface,
+    pricingCachingInterface?: CachingInterface
+  ) {
+    // check if we have the all pricing list in the static variable
+    let missingPricingList = AwsCatalogClient.getMissingPricingList(resources)
```
Comment (marashlyansuren) [inline src/adapters/aws/clients/aws-catalog-client.ts:26]: hmm, but is this static?

Comment (sergeymirzoyan1991) [reply src/adapters/aws/clients/aws-catalog-client.ts:26]: what you mean?

Comment (marashlyansuren) [reply src/adapters/aws/clients/aws-catalog-client.ts:26]: `check if we have the all pricing list in the static variable` I am about your comment variable

Comment (sergeymirzoyan1991) [reply src/adapters/aws/clients/aws-catalog-client.ts:26]: ah, yes we have a static variable that keeps the pricing info

**Review by marashlyansuren — APPROVED** [review]

**Pull request merged**